.infrastructure/mysql/init-configMap.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mysql-init-script
+  namespace: mysql
+data:
+  init.sql: |
+    CREATE USER 'mysqluser'@'%' IDENTIFIED BY 'P@ssw0rd';
```

Comment:
Hardcoding passwords in configuration files is a security risk. Consider using Kubernetes Secrets to manage sensitive information like database passwords.